README.md

```diff
@@ -1,15 +1,17 @@
-# mcuboot
+# [mcuboot](http://mcuboot.com/)
 
-This is mcuboot, version 0.9.0
```

Comment:
I don't think we should remove the version number from the README. It makes it a lot easier to find the version number and how it relates to the commits.

Reply:
Added back.